benchmark_config:
  parameters:
    name: "PlanningPipeline"
    # Number of runs per configuration
    runs: 25
    # Timeout in seconds
    timeout: 10.0
  collision_detectors:
    - FCL
    - Bullet
  planning_pipelines:
    - name: ompl
      planners:
        - RRTConnectkConfigDefault
        - RRTkConfigDefault
        - RRTstarkConfigDefault
        - ESTkConfigDefault
        - PRMkConfigDefault
        - LBKPIECEkConfigDefault
        - BiESTkConfigDefault
